engine_modules/corporation/models.py

```diff
@@ -93,19 +92,48 @@ class Meta:
 	base_corporation_slug = models.CharField(max_length=20)
 	game = models.ForeignKey(Game)
 	assets = models.SmallIntegerField()
-	historic_market = models.ForeignKey(Market)
+	market_assets = models.SmallIntegerField()
```

Comment:
Can you explain the difference between assets and market_assets and market_modifier in a comment?